linux_os/guide/system/software/sudo/sudoers_validate_passwd/tests/sudoers_validate_passwd_pass.sh

```diff
@@ -0,0 +1,6 @@
+# platform = SUSE Linux Enterprise 15
```

Comment:
Please rename this file so that it ends in .pass.sh

Reply:
done